org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/model/edit/ITextualMultiModification.java

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2019 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.ui.editor.model.edit;
+
+import com.google.common.annotations.Beta;
+
+/**
+ * Specialization of {@link IModification} the Author claims to be executable as Multi-Quickfix.
```

Comment:
Even though I do like the co-notation of `the author claims to be ...`, I think 
`Specialization of {@link IModification textual modifications} that can be applied as a multi quick-fixes` is a little nicer.